### PR TITLE
Added a locale dropdown markup

### PR DESCRIPTION
**Pull request by flukeout**
Added it both the homepage and the editor.

@gideonthomas R?

Right now, the same markup lives in two separate spots, in both ``userbar.html`` files, one for the editor and one for the editor. Let's move it to a separate file and include it somehow? If you have ideas on how to do that let me know.

Should look like this...

image

### CODE REVIEW

**Comment by gideonthomas** [comment]
@flukeout looks good to me! I would like to make it functional first before landing it on staging since the Pontoon stuff is underway. Let me know if you would like to do this (I can help figure out how) or I can take a shot at it.

As for the `userbar.html` file, can you file that? I can refactor it into one file when I get a little bit of time (I suspect we will have a few minor issues with pathing).

Also, I would love to learn Bortugese! Sounds bery bad bass!

**Comment by flukeout** [comment]
Ok @gideonthomas - give this a try, it should...

* Show the selected language name in the green header
* Show a checkmark beside the selected language
* Send both the editor & homepage to the correct URL when the locale is changed via the dropdown

File: views/homepage/userbar.html [hunk 1]
```diff
@@ -10,37 +10,59 @@
...
+        <div class="dropdown">
+          <div class="dropdown-toggle">
+            {{ languages[locale].nativeName }}
+            <img class="icon-arrow-down" src="/img/icon/dropdown-arrow.svg">
+          </div>
+          <div class="dropdown-content">
+            <ul class="language-list">
+
+              {% for langCode, obj in languages %}
+                <li><a {% if langCode == locale %} class="selected" {% endif %} href="{{ URL_PATHNAME }}{{ langCode }}/">{{ obj.nativeName }}</a></li>
```
Comment (gideonthomas) [inline views/homepage/userbar.html]: this will be `/{{ langCode }}{{ URL_PATHNAME }}`

File: views/homepage/userbar.html [hunk 2]
```diff
@@ -10,37 +10,59 @@
...
+        <div class="dropdown">
+          <div class="dropdown-toggle">
+            {{ languages[locale].nativeName }}
+            <img class="icon-arrow-down" src="/img/icon/dropdown-arrow.svg">
+          </div>
+          <div class="dropdown-content">
+            <ul class="language-list">
+
+              {% for langCode, obj in languages %}
+                <li><a {% if langCode == locale %} class="selected" {% endif %} href="/{{ langCode }}{{ URL_PATHNAME }}">{{ obj.nativeName }}</a></li>
```
Comment (gideonthomas) [inline views/homepage/userbar.html]: Probably add a `target="_self"` here

**Comment by flukeout** [comment]
Also, need to fix the styling here...

image

**Comment by flukeout** [comment]
Fixed the missing right padding and added a target=_self for the homepage & projects page locale dropdown.

cc @gideonthomas

**Comment by gideonthomas** [comment]
I merged this in

**Pull request closed without merging**